vscode-extensions/snippets/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+We greatly welcome contributions. You can help make the snippets better by improving snippets, documentation, samples.
+
+## Developing
+
+Install vsce globally:
+```bash
+npm install --global @vscode/vsce
+```
+
+Verify the installation:
+```bash
+vsce --version
+```
+
+* You can run and debug this extension by using VS Code menu `Run and Debug` on the left sidebar or Press `F5`
+* Create a new file with a file name suffix matching your language.
+* Verify that your snippets are proposed on IntelliSense.
+
+### Make changes
+
+* You can relaunch the extension from the debug toolbar after making changes to the files listed above.
+* You can also reload (`Ctrl+R` or `Cmd+R` on Mac) the VS Code window with your extension to load your changes.
+
+## Install extension
+
+* To start using your extension with Visual Studio Code by use command `npm run install:extension`
+
+## CLI Commands
+
+Package extension to `*.vsix` file:
+```bash
+npm run package
```

Comment:
Please update CLI commands to align with CLI in package.json

Reply:
Updated